<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataLoad</class>
 <widget class="QDialog" name="DataLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Parse filename</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Path inside file</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="parseFnameLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter Keys</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="fileTypeComboBox">
       <item>
        <property name="text">
         <string>Matlab (.mat)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HDF5 (.h5)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select datatype</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="pathRecursiveCheckBox">
       <property name="text">
        <string>Find files recursively</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pathDataButton">
       <property name="text">
        <string>Data Path...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="fileStructureLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pathFilterKeysLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pathDataLineEdit"/>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="dataLoadCancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dataLoadOkButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="loadDataProgressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
